packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/converters.tests.js

```diff
@@ -1,7 +1,11 @@
 import $ from 'jquery';
 
+import 'ui/html_editor';
```

Comment:
I believe these tests should be placed not here while `converter` is just an HtmlEditor's option, not a real converter